R/init.R

```diff
@@ -32,11 +32,11 @@
 #'   This parameter is deprecated. Use `modify_footer()` on the teal app object instead.
 #' @param id `r lifecycle::badge("deprecated")` (`character`) Optionally,
 #'   a string specifying the `shiny` module id in cases it is used as a `shiny` module
-#'   rather than a standalone `shiny` app. This is a legacy feature. Deprecated since v0.15.3
+#'   rather than a standalone `shiny` app. This is a legacy feature. Deprecated since v0.16.0
```

Comment:
We already have the deprecated badge. Do we have to add that statement additionally? Do we have to specify the version? What's the convention?

Reply:
I agree, we don't need to repeat the deprecated version as this will also been shown in the `deprecate_soft` message

Reply:
Updated in d23ad65a18b240268723c43f32df6a17f91d3a57